batch/batch/front_end/front_end.py

```diff
@@ -534,10 +575,9 @@ async def _query_batches(request, user):
 
 @routes.get('/api/v1alpha/batches')
 @prom_async_time(REQUEST_TIME_GET_BATCHES)
-@rest_authenticated_users_only
-async def get_batches(request, userdata):
-    user = userdata['username']
-    batches, last_batch_id = await _query_batches(request, user)
+@rest_billing_project_users_only
```

Comment:
This request doesn't have a batch_id, right?

Reply:
No. It's broken. See the comment I made above about making the default query string `f'user:{user}'`